From 677fa78620e762528db0aa0609c22d15b75352bf Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Thu, 17 Dec 2020 09:16:46 +1000
Subject: [PATCH libwacom] meson.build: fix typo causing the data file test to
 be installed

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 6a38b36..0d4e346 100644
--- a/meson.build
+++ b/meson.build
@@ -107,7 +107,7 @@ install_subdir('data',
 	       exclude_files: ['wacom.example',
 			       'check-files-in-git.sh',
 			       'check-svg-exists.sh',
-			       'test-data_files.py',
+			       'test_data_files.py',
 			       'Makefile.am',
 			       'Makefile.in',
 			       'layouts/Makefile.am',
-- 
2.28.0

