From 7fe5e50ac7fcd08e042d8221bb9c479c3c280d3e Mon Sep 17 00:00:00 2001
From: Maximilian Luz <luzmaximilian@gmail.com>
Date: Sun, 20 Apr 2025 00:21:02 +0200
Subject: [PATCH] Revert minimum meson version to v0.53.0

---
 meson.build | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/meson.build b/meson.build
index 9c0b1fbc..8a1de43c 100644
--- a/meson.build
+++ b/meson.build
@@ -2,7 +2,7 @@ project('libwacom', 'c',
 	version: '2.16.1',
 	license: 'HPND',
 	default_options: [ 'c_std=gnu99', 'warning_level=2' ],
-	meson_version: '>= 0.57.0')
+	meson_version: '>= 0.53.0')
 
 dir_bin      = get_option('prefix') / get_option('bindir')
 dir_data     = get_option('prefix') / get_option('datadir') / 'libwacom'
@@ -179,7 +179,7 @@ testdevices = {
 }
 foreach name : testdevices.keys()
 	match = testdevices[name]
-	test(f'debug-device-@name@',
+	test('debug-device-@0@'.format(name),
 	debug_device,
 	args: [match],
 	suite: ['all'])
@@ -295,7 +295,6 @@ if get_option('tests').enabled()
 	if valgrind.found()
 		valgrind_suppressions_file = dir_test / 'valgrind.suppressions'
 		add_test_setup('valgrind',
-			       exclude_suites: ['not-in-valgrind'],
 			       exe_wrapper: [valgrind,
 					     '--leak-check=full',
 					     '--gen-suppressions=all',
@@ -325,7 +324,7 @@ if get_option('tests').enabled()
 			     args: ['--verbose',
 				     '-rfES',
 				     '--log-level=DEBUG',
-				     '--log-file', meson.project_build_root() / 'meson-logs' / 'pytest.log',
+				     '--log-file', meson.build_root() / 'meson-logs' / 'pytest.log',
 				     '--log-file-level=DEBUG',
 				     meson.current_source_dir() / 'test' / f,
 			     ],
-- 
2.50.0

